### PR TITLE
[Tensorflow][ECS][EKS][Test] Add albert model to inference test

### PR DESCRIPTION
**Pull request by Teng-xu**
I've prepended PR tag with frameworks/job this applies to : [tensorflow] | [test] | [ecs, eks]

When creating a pr:

    [ ] I've modified src/config/build_config.py in my PR branch by setting ENABLE_EI_MODE = True

When PR is reviewed and ready to be merged:

    [ ] I've reverted the code change on the config file mentioned above

For reviewer, before merging, please cross-check:

    [ ] I've verified the code change on the config file mentioned above has already been reverted

Description:
Replacing the model for ecs/eks tensorflow inference test.
Albert model link: https://github.com/aws-samples/deep-learning-models/tree/master/models/nlp/albert

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/dlc_tests/eks/tensorflow/inference/test_eks_tensorflow_inference.py [hunk 1]
```diff
@@ -14,13 +14,13 @@ def test_eks_tensorflow_half_plus_two_inference(tensorflow_inference):
 
     processor = "gpu" if "gpu" in tensorflow_inference else "cpu"
 
-    model_name = f"saved_model_half_plus_two_{processor}"
+    model_name = f"albert"
```
Comment (arjkesh) [inline test/dlc_tests/eks/tensorflow/inference/test_eks_tensorflow_inference.py:17]: the name of this function is test_eks_half_plus_two_inference -- do you want to change it completely? or add a new bert model test?

File: test/test_utils/ecs.py [hunk 2]
```diff
@@ -27,7 +27,7 @@
 
 ECS_INSTANCE_ROLE_ARN = "arn:aws:iam::669063966089:instance-profile/ecsInstanceRole"
 ECS_INSTANCE_ROLE_NAME = "ecsInstanceRole"
-TENSORFLOW_MODELS_BUCKET = "s3://tensoflow-trained-models"
+TENSORFLOW_MODELS_BUCKET = "s3://aws-tensorflow-dlc-models"
```
Comment (arjkesh) [inline test/test_utils/ecs.py:30]: should this also be changed? I don't see any ecs test changes

Comment (arjkesh) [inline test/test_utils/ecs.py:30]: This models bucket may be used for other models as well -- did you sync everything from the original bucket? if not, you could get some unexpected failures

File: test/dlc_tests/eks/tensorflow/inference/test_eks_tensorflow_inference.py (hunk 1 above)
Comment (TusharKanekiDey) [reply test/dlc_tests/eks/tensorflow/inference/test_eks_tensorflow_inference.py:17]: +1

File: test/dlc_tests/ecs/tensorflow/inference/test_ecs_tensorflow_inference.py [hunk 3]
```diff
@@ -51,23 +51,23 @@ def test_ecs_tensorflow_inference_eia(tensorflow_inference_eia, ecs_container_in
...
+    model_name = "albert"
     service_name = task_family = revision = None
     try:
         service_name, task_family, revision = ecs_utils.setup_ecs_inference_service(
             tensorflow_inference, "tensorflow", ecs_cluster_arn, model_name, worker_instance_id, num_gpus=num_gpus,
             region=region
         )
         model_name = get_tensorflow_model_name("gpu", model_name)
-        inference_result = request_tensorflow_inference(model_name, ip_address=public_ip_address)
+        inference_result = request_tensorflow_inference_nlp(model_name, ip_address=public_ip_address)
```
Comment (YYStreet) [inline test/dlc_tests/ecs/tensorflow/inference/test_ecs_tensorflow_inference.py:70]: Could you retain the existing test and add the new model? Feel free to introduce another test if necessary

Comment (Teng-xu) [reply test/dlc_tests/ecs/tensorflow/inference/test_ecs_tensorflow_inference.py:113]: Retained the existing tests

**Comment by YYStreet** [comment]
Updated the version number of this PR to tf2.3 https://github.com/aws/deep-learning-containers/commit/c49e51a634248773d3fbd429fb4acdc5e3067c80

**Comment by tejaschumbalkar** [comment]
Failures seen in sanity check:
1. test_safety_check - Will be fixed by #559 
2. test_generate_coverage_doc - @Teng-xu can you take a look

File: test/dlc_tests/eks/tensorflow/inference/test_eks_tensorflow_inference.py [hunk 4]
```diff
@@ -9,8 +9,8 @@
 import test.test_utils as test_utils
 
 
-@pytest.mark.model("half_plus_two")
-def test_eks_tensorflow_half_plus_two_inference(tensorflow_inference):
+@pytest.mark.model("albert")
```
Comment (arjkesh) [inline test/dlc_tests/eks/tensorflow/inference/test_eks_tensorflow_inference.py:12]: any reason we are removing half plus two completely?

Comment (Teng-xu) [reply test/dlc_tests/eks/tensorflow/inference/test_eks_tensorflow_inference.py:12]: > any reason we are removing half plus two completely?

we want to include more models for testing, and we already have ecs testing on half_plus_two

**Review by SergTogul — APPROVED** [review]

**Pull request merged**